.github/workflows/pr-test.yml

```
@@ -644,6 +677,104 @@ jobs:
       - name: Run JIT Unittest Part 3 (T4)
         run: bash ci/bash.sh ${DOCKER_IMAGE} ./scripts/task_jit_run_tests_part3.sh
 
+  # ---------------------------------------------------------------------------
+  # GPU JIT Tests - B200 (Blackwell) - Capacity Block
+  # Triggers dynamic CB purchase via EventBridge → CB Manager Lambda
+  # ---------------------------------------------------------------------------
+  gpu-tests-b200:
+    name: JIT Unittest (B200)
+    needs: setup
+    if: needs.setup.outputs.skip_build != 'true' && github.event.inputs.skip_gpu != 'true'
+    runs-on: [self-hosted, Linux, X64, gpu, b200]
```

actionlint (1.7.10)

[error] label "gpu" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)